sp/src/game/client/flashlighteffect.cpp

```diff
@@ -103,7 +110,7 @@ CFlashlightEffect::CFlashlightEffect(int nEntIndex)
 
 #ifdef EZ
 	m_iFlashLightType = type;
-	if ( g_pMaterialSystemHardwareConfig->SupportsBorderColor() && IsNVG() )
+	if ( g_pMaterialSystemHardwareConfig->SupportsBorderColor() && !r_nvg_force_nonbordered.GetBool() && IsNVG() )
```

Comment:
I'm a little confused by this line to be honest. Does this change behavior? It seems like before this condition would be true, but not it will be false. Unless supports border color was already false?